### PR TITLE
EDGE-631 Add SIP URI Transfer in webrtc sample apps

### PR DESCRIPTION
**Pull request by carrington-bw**
I still need to update the Java app but I got Node working

### CODE REVIEW

File: nodejs/webrtc-hello-world/src/server.ts [hunk 1]
```diff
@@ -282,11 +281,8 @@ const callPhone = async (phoneNumber: string, participant: Participant) => {
...
  */
 const generateTransferBxml = async (deviceToken: string) => {
-  //Get the tid out of the participant jwt
-  var decoded: any = jwt_decode(deviceToken);
-
-  return `<?xml version="1.0" encoding="UTF-8" ?>
-    <Response>
-      <Transfer transferCallerId="${decoded.tid}"><PhoneNumber>${sipxNumber}</PhoneNumber></Transfer>
-    </Response>`;
+  return '<?xml version="1.0" encoding="UTF-8" ?>\n'
```
Comment (carrington-bw) [inline nodejs/webrtc-hello-world/src/server.ts:284]: I tried including the WebRTC SDK in order to use the JS helper but it doesn't include a declarations file and thus won't work with Typescript. I'll work with DevX on getting that fixed but in the meantime I elected to include the XML directly.

File: nodejs/webrtc-hello-world/package.json [hunk 2]
```diff
@@ -12,13 +12,13 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@bandwidth/webrtc": "^0.4.1",
     "@types/express": "^4.17.2",
-    "@types/jwt-decode": "^2.2.1",
+    "@types/node": "^14.14.14",
```
Comment (carrington-bw) [inline nodejs/webrtc-hello-world/package.json:17]: Had to install this in order to use `require()` in our Typescript server to pull in the browser SDK

File: nodejs/webrtc-hello-world/src/server.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import dotenv from "dotenv";
 import express from "express";
 import bodyParser from "body-parser";
 import axios from "axios";
-import jwt_decode from "jwt-decode";
+const bandwidthWebRTC = require("@bandwidth/webrtc");
```
Comment (carrington-bw) [inline nodejs/webrtc-hello-world/src/server.ts:6]: Definitely look at this file and how I did it. I'm not up to speed on JavaScript or Typescript conventions.

File: java/webrtc-hello-world/src/main/java/com/bandwidth/webrtc/examples/helloworld/HelloWorldController.java [hunk 4]
```diff
@@ -130,7 +123,7 @@ public HelloWorldController(AccountProperties accountProperties, VoiceProperties
         calls.put(callId, response);
 
         // Generate transfer BXML from the device token and return that to the Voice API
-        String transferBxml = generateTransferBxml(response.getToken());
+        String transferBxml = WebRtcTransfer.generateBxml(response.getToken());
```
Comment (carrington-bw) [inline java/webrtc-hello-world/src/main/java/com/bandwidth/webrtc/examples/helloworld/HelloWorldController.java:126]: Reminder to update this to use generateBxmlVerb() as well as update pom.xml with new Java SDK version

File: nodejs/webrtc-hello-world/src/server.ts [hunk 5]
```diff
@@ -17,9 +17,9 @@ const voiceApplicationPhoneNumber = <string>process.env.VOICE_APPLICATION_PHONE_
 const voiceApplicationId = <string>process.env.VOICE_APPLICATION_ID;
 const voiceCallbackUrl = <string>process.env.VOICE_CALLBACK_URL;
 const outboundPhoneNumber = <string>process.env.OUTBOUND_PHONE_NUMBER;
-
 const callControlUrl = `${process.env.BANDWIDTH_WEBRTC_CALL_CONTROL_URL}/accounts/${accountId}`;
-const sipxNumber = <string>process.env.BANDWIDTH_WEBRTC_SIPX_PHONE_NUMBER;
+
+var webRTCController = bandwidthWebRTC.APIController;
```
Comment (cejohnson-bw) [inline nodejs/webrtc-hello-world/src/server.ts:22]: Any reason this is a `var` vs a `const` or at least a `let`? Also purely preference, but I think I'd prefer `Rtc` capitalization to `RTC`.

File: nodejs/webrtc-hello-world/src/server.ts (hunk 3 above)
Comment (cejohnson-bw) [reply nodejs/webrtc-hello-world/src/server.ts:6]: So I guess you can't use `import` here? I think I ran into the same thing when I looked into it.

**Review by cejohnson-bw — APPROVED** [review]
Very minor comments, overall LGTM. Guess now that we're importing the node SDK we should come back around and replace the HTTP calls.

**Review by psherk-bw — APPROVED** [review]
Once Chris J's comments are resolved I'm good to go

File: nodejs/webrtc-hello-world/src/server.ts (hunk 3 above)
Comment (carrington-bw) [reply nodejs/webrtc-hello-world/src/server.ts:6]: Yeah, you can only use import when you have a type declarations file (.d.ts). It's a typescript thing. For packages without it you can use require plus the types/node package.

File: nodejs/webrtc-hello-world/src/server.ts (hunk 5 above)
Comment (carrington-bw) [reply nodejs/webrtc-hello-world/src/server.ts:22]: I lifted the code from another app, so no reason. I'll update.

**Pull request merged**